### PR TITLE
use the library version and update patch version to latest available

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

File: cluster/manifests/skipper/skipper-redis.yaml [hunk 1]
```diff
@@ -42,7 +42,7 @@ spec:
                 - skipper-ingress-redis
       priorityClassName: "{{ .Cluster.ConfigItems.system_priority_class }}"
       containers:
-      - image: registry.opensource.zalan.do/zmon/redis:6-master-12
+      - image: registry.opensource.zalan.do/library/redis-6-alpine:6-alpine-20210712
         name: skipper-ingress-redis
```
Comment (mikkeloscar) [inline cluster/manifests/skipper/skipper-redis.yaml:46]: The zmon image runs with the flag `--save ""` which to my understanding means to disable saving to disk: https://raw.githubusercontent.com/redis/redis/6.0/redis.conf

For this reason we probably want to explicitly set this flag via `args` to run redis in the same way with the new image.

Comment (szuecs) [reply cluster/manifests/skipper/skipper-redis.yaml:46]: where do you see that they use this flag?
Is it in the script they have in the docker container? 
I could not easily cat it at runtime so I thought this is fine. Honestly quite unreasonable to have "library" with completely 0 information about the defaults.

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**